portfolio/src/main/webapp/script.js

```diff
@@ -30,18 +30,37 @@ function addRandomGreeting() {
 /**
  * Adds a personal greeting to the page.
  */
-async function getPersonalHello() {
-  const name = document.getElementById('name-input').value;
-  const data = { name : name };
-
-  const response = await fetch('/data', {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify(data),
-  });
-  const text = await response.text();
-  console.log(text);
-  document.getElementById('personal-hello-containter').innerText = text;
+// async function getPersonalHello() {
```

Comment:
In general, try not to commit commented code.

Reply:
Fixed!